src/components/ui/Callout/fragments/CalloutRoot.tsx

```
@@ -7,13 +7,29 @@ type CalloutRootProps = {
     children?: React.ReactNode;
     className?: string | '' ;
     color?: string;
+    variant?: string;
+    size?: string;
     customRootClass?: string;
     props?: Record<any, any>[]
 }
 
-const CalloutRoot = ({ children, className, color, customRootClass, ...props }: CalloutRootProps) => {
+const CalloutRoot = ({ children, className, color, variant='', size='', customRootClass, ...props }: CalloutRootProps) => {
```

ESLint

[error] Operator '=' must be spaced.

(space-infix-ops)